src/test/java/software/amazon/awssdk/crt/test/HttpClientConnectionManagerTest.java

```diff
@@ -5,11 +5,7 @@
 import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.concurrent.Callable;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
+import java.util.concurrent.*;
```

Comment:
Very minor and probably a result of IDE settings, but I think we have a preference for keeping imports explicit and not wild-carded.

Reply:
yup IDE auto import settings, will fix